contractcourt/chain_watcher_test.go

```diff
@@ -575,3 +578,173 @@ func TestChainWatcherLocalForceCloseDetect(t *testing.T) {
 		})
 	}
 }
+
+// TestCooperativeCloseInitiator tests that the chain arbitrator will correctly
+// record the close type of cooperatively closed channels. This test covers the
+// case where a channel is marked with the legacy status ChanStatusCoopBroadcast
+// and the
+func TestCooperativeCloseInitiator(t *testing.T) {
+	tests := []struct {
+		name      string
+		closeType channeldb.ClosureType
+	}{
+		{
+			name:      "legacy cooperative close",
+			closeType: channeldb.CooperativeClose,
+		},
+		{
+			name:      "local init cooperative close",
+			closeType: channeldb.LocalCoopClose,
+		},
+		{
+			name:      "remote init cooperative close",
+			closeType: channeldb.RemoteCoopClose,
+		},
+	}
+
+	for _, test := range tests {
+		test := test
```

Comment:
nit: remove

Reply:
this is required for the linter (goscope) to address concurrency issues with [parallel tests](https://github.com/golang/go/issues/16586#issuecomment-237355153)